### PR TITLE
Add jquery.smoothState.js

### PR DESCRIPTION
**Pull request by ApfelUser**
Unobtrusive page transitions with jQuery.

**[smoothstate.com](http://smoothstate.com/)**
**[github:miguel-perez/smoothState.js](https://github.com/miguel-perez/smoothState.js)**

### CODE REVIEW

File: files/jquery.smoothstate/update.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "npm",
+  "name": "smoothstate",
+  "repo": "miguel-perez/smoothState.js",
+  "files": {
+    "include": ["jquery.smoothState.min.js", "src/jquery.smoothState.js"]
```
Comment (jimaek) [inline files/jquery.smoothstate/update.json]: I dont think this is needed in production src/jquery.smoothState.js
We can remove it

Comment (ApfelUser) [inline files/jquery.smoothstate/update.json]: Hm. I do check at my NodeJS app whether the app is in a production environment, and if yes I just append `.min` to my CSS and JS files. This is so I can debug easier when developing. I also saw that lots of libraries on JSDelivr also provide the unminified files, so I thought its normal. But I'll remove it.

**Comment by jimaek** [comment]
@jsdelivrbot squash and merge

**Comment by jimaek** [comment]
Thank you

**Comment by ApfelUser** [comment]
Thanks, but actually I have to thank you for this awesome service! :+1: :smile:

**Comment by jimaek** [comment]
:)

**Pull request closed without merging**